### ISSUE TITLE
Playback Speed Indicator : Elements must have sufficient color contrast

### ISSUE THREAD

**Issue by davefollett** [codingblocks/podcast-app#231]
## 💥 Impact rating
serious

## 👣 Steps to reproduce
Add a podcast to the queue, start playing it, pause it, and then [run the aXe Accessibility Test](https://github.com/codingblocks/podcast-app/wiki/Accessibility-(a11y)-Testing#run-axe-accessibility-test).

## ⚛️ Element(s) with issue
Element location
`.sc-ckVGcZ`

Element source
`<button class="sc-ckVGcZ iEfsnd">1x</button>`

## 🛠️ Recommended fix
Fix the following:
Element has insufficient color contrast of 3.77 (foreground color: #359189, background color: #ffffff, font size: 12.0pt, font weight: normal). Expected contrast ratio of 4.5:1

Related node:
`.sc-ckVGcZ`

## 📚 Reference information
https://dequeuniversity.com/rules/axe/3.0/color-contrast?application=AxeChrome
